CITATIONS.md

```diff
@@ -14,6 +14,14 @@
 
```

Comment:
I know you've added your flexiformatter tool which is a public github repo, might be worth adding the link to the repo here. We've got a couple tools like that ( e.g. pigz)